test/e2e/utils/utils.go

```diff
@@ -118,17 +119,83 @@ func LocateRepoFile(repopath string) (string, error) {
 	return "", errors.New("no file found, try to define PLUGINS_REPO_DIR pointing to the root of the repository")
 }
 
-// CreateKustomizationOverlay creates an overlay with overridden namespace.
-func CreateKustomizationOverlay(namespace, base, overlay string) error {
-	relPath := ""
-	for range strings.Split(overlay[1:], "/") {
-		relPath = relPath + "../"
+func copyFiles(srcDir, dstDir string) error {
+	err := filepath.WalkDir(srcDir, func(path string, d os.DirEntry, err error) error {
+		if d.IsDir() || err != nil {
+			return nil
+		}
+
+		n, err := os.ReadFile(path)
+		if err != nil && err != io.EOF || len(n) == 0 {
+			return err
+		}
+
+		fn := filepath.Join(dstDir, filepath.Base(path))
+
+		if err := os.WriteFile(fn, n, 0600); err != nil {
+			return err
+		}
+
+		return nil
+	})
+
+	return err
+}
+
+// CreateKustomizationOverlay copies the base overlay, and changes the namespace
```

Comment:
perhaps add a comment that any cleanup after copying is left to the caller?

Reply:
Added